app/connector/aws-sqs/src/main/java/io/syndesis/connector/aws/sqs/AWSSQSBatchMessagesCustomizer.java

```diff
@@ -49,6 +46,5 @@ public void beforeProducer(final Exchange exchange) throws IOException {
             }
         }
         in.setBody(messageList);
-        in.setHeader(SqsConstants.SQS_OPERATION, SqsOperations.sendBatchMessage);
```

Comment:
Had a live conversation already about this, it seems it may be needed.

Reply:
Yes that is a problem indeed. Thanks for pointing to it! We are missing a backport to camel-fuse 2.23.x here. I have opened an issue to do the backport: https://issues.jboss.org/browse/ENTESB-12348

This is the commit that needs to be backported: https://github.com/jboss-fuse/camel/commit/53670be2a404b9c258897d02019892a36ba46664#diff-5b2805df49a4af06496342bb13db80a3

Once this is done we can add the header again in our code. I will add a comment in our code and reference the backporting issue.